### DISCUSSION TITLE
Total power is less than an individual circuit;  Where to look?

### DISCUSSION THREAD

**Discussion by catatung** [emporia-vue-local/esphome#183] [General]
Hello,

I've got this set up for awhile now and just recently added the main CT clamps for both phases.  Now that I have this, I have a reading for Phase A power, Phase B power and total power.

I've recently noticed that sometimes, Total power is reading less than that of one single circuit.  This seems impossible to me -- if one specific circuit is drawing 540.0W, how can total power be less (513.3W?)  I assume this disparity is because of a difference between adding up all circuit CT readings vs adding just the two main phase CT sensors, but how do I determine which is accurate and where should I look to make corrections? 

Thanks for any help or guidance

**Comment by flaviut** [comment]
Yup, this sounds like error between the various clamps. You need a calibration reference: either a calibrated way of measuring current, or a known fixed load. Clamp current meters are a common way of measuring current in this application, and some of the more expensive ones will come with lab calibration papers.

See also:

- https://github.com/emporia-vue-local/esphome/discussions/131#discussioncomment-5106009
- https://github.com/emporia-vue-local/esphome/discussions/182#discussion-5397963
